spec/services/service_provider_seeder_spec.rb

```diff
@@ -150,7 +150,7 @@
               acs_url: 'http://example.com/test/saml/decode_assertion',
               assertion_consumer_logout_service_url: 'http://example.com/test/saml/decode_slo_request',
               block_encryption: 'aes256-cbc',
-              cert: 'saml_test_sp',
+              cert: ['saml_test_sp'],
```

Comment:
**Question:** Was this meant to change to `certs`?
```suggestion
              certs: ['saml_test_sp'],
```

Reply:
great catch! now I'm worried why the specs passed